workflows/genome_annotation.nf

```diff
@@ -0,0 +1,148 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    VALIDATE INPUTS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
+
+// Validate input parameters
+WorkflowSeqqc.initialise(params, log)
+
+// TODO nf-core: Add all file path parameters for the pipeline to the list below
+// Check input path parameters to see if they exist
+def checkPathParamList = [  params.multiqc_config, params.fasta ]
+for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
+
+// Check mandatory parameters
+if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
+
+/*
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    IMPORT LOCAL MODULES/SUBWORKFLOWS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+//
+// SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
+//
+include { INPUT_CHECK           } from '../subworkflows/local/input_check'
+include { GENE_ANNOTATION       } from '../subworkflows/local/gene_annotation'
+include { PHAGE                 } from '../subworkflows/local/phage'
+include { PANGENOME_ANALYSIS    } from '../subworkflows/local/pangenome_analysis'
+include { PLASMIDS              } from '../subworkflows/local/plasmids'
+include { AMR_ANNOTATION        } from '../subworkflows/local/amr_annotation'
+
```

Comment:
Missing CRISPR subworkflow, but that's because I completely forgot to open a PR to merge the subworkflow! The subworkflow is ready to be merged [39-create-a-sub-workflow-for-crispr-prediction-and-typing](/cidgoh/nf-seqqc/tree/39-create-a-sub-workflow-for-crispr-prediction-and-typing). I'm not too bothered by this, as we can always add this in the next release

Reply:
Fixed with [15b8ff0](https://github.com/cidgoh/nf-seqqc/pull/58/commits/15b8ff09ed6b836e659308f8a1856b5d88d6aaed)